public/app/features/search/page/components/ActionRow.tsx

```diff
@@ -0,0 +1,95 @@
+import { css } from '@emotion/css';
+import React, { FC, FormEvent } from 'react';
+
+import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { HorizontalGroup, RadioButtonGroup, useStyles2, Checkbox, Button } from '@grafana/ui';
+import { SortPicker } from 'app/core/components/Select/SortPicker';
+import { TagFilter, TermCount } from 'app/core/components/TagFilter/TagFilter';
+
+import { DashboardQuery, SearchLayout } from '../../types';
+
+export const layoutOptions = [
+  { value: SearchLayout.Folders, icon: 'folder', ariaLabel: 'View by folders' },
+  { value: SearchLayout.List, icon: 'list-ul', ariaLabel: 'View as list' },
+];
+
+if (config.featureToggles.dashboardPreviews) {
```

Comment:
@axelavargas -- this file is mostly a copy of the other ActionRow except that it has:
1. additional grid layout
2. shows the selected datasource
3. the tags that are provided are a subset of query results (not all tags)
4. useStyle2